recipes/falco/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.2.1" %}
+
+package:
+  name: falco
+  version: {{ version }}
+
+build:
+  number: 1
```

Comment:
please start with 0

Reply:
Hi @bgruening

I had a pr accepted yesterday that started with 0 so this is the next build. My first commit failed the tests stating I had to increase the number. What should I do for the lint test to pass with build 0?

Reply:
Ah, from the PR it looks like falco is new, which was the cause of the confusion. You might want to pull the master branch from bioconda next time before making changes :)

Reply:
understood, and sorry again! Still getting the hang of making conda builds. Thank you so much for accepting the PR so quickly!